de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java

```diff
@@ -226,6 +234,57 @@ public Response getCa(@PathParam("caUuid") String caUuid) {
 		public Response putCa(ABeanCategoryAssignment bean) {
 			return Response.status(Response.Status.OK).build();
 		}
+		
+		/**
+		 * Returns a response with a list of the root seis
+		 * @return a server response
+		 */
+		@GET
+		@Path(ROOT_SEIS)
+		@Produces(MediaType.APPLICATION_JSON)
+		public Response getRootSeis() {
+			try {
+				EList<StructuralElementInstance> rootSeis = repository.getRootEntities();

```

Comment:
Can we just use "List" on the left hand side?

Reply:
Sure